src/libs/actions/Policy.ts

```diff
@@ -3649,7 +3650,9 @@ function clearPolicyTagErrors(policyID: string, tagName: string) {
 
 function renamePolicyTag(policyID: string, policyTag: {oldName: string; newName: string}) {
     const tagListName = Object.keys(allPolicyTags?.[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`] ?? {})[0];
-    const oldTag = allPolicyTags?.[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`]?.[tagListName]?.tags?.[policyTag.oldName] ?? {};
+    const oldTagName = PolicyUtils.escapeTagName(policyTag.oldName);
```

Comment:
@bernhardoj  Why do we need to escape the old tag?

Reply:
because we escaped the tag name when creating, It means that the old tag is escaped

Reply:
The tag name that we pass to the old name is the "cleaned" tag name.
https://github.com/Expensify/App/blob/e8ae3c5acedf0e6788dc574c7b6f3043ca37092a/src/pages/workspace/tags/EditTagPage.tsx#L38
https://github.com/Expensify/App/blob/e8ae3c5acedf0e6788dc574c7b6f3043ca37092a/src/pages/workspace/tags/EditTagPage.tsx#L61

Reply:
@bernhardoj Should we pass route.params.tagName instead of currentTagName

https://github.com/Expensify/App/blob/e8ae3c5acedf0e6788dc574c7b6f3043ca37092a/src/pages/workspace/tags/EditTagPage.tsx#L61

Reply:
I would prefer to keep it as it is. Having 2 different sources of the tag name, `route.params.tagName` and `currentTagName` makes it more confusing, IMO.

Reply:
But we shouldn't make a redundant logic. Currently, we cleaned tag name and escape again

Reply:
IMO, we only need to escape new tag

Reply:
Ok, I have updated it.